tools/vsimporter/xib2nib/UILabel.cpp

```diff
@@ -188,14 +184,12 @@ void UILabel::ConvertStaticMappings(NIBWriter* writer, XIBObject* obj) {
         AddInt(writer, "UIBaselineAdjustment", _baselineAdjustment);
     }
 
-    if (_numberOfLines == 1) {
-        if (_adjustsFontSizeToFit) {
-            AddBool(writer, "UIAdjustsFontSizeToFit", _adjustsFontSizeToFit);
-        }
-
-        if (_minimumFontSize != -1.0f) {
-            AddOutputMember(writer, "UIMinimumFontSize", new XIBObjectFloat(_minimumFontSize));
-        }
+    if (_minimumFontSize != -1.0f) {
+        AddOutputMember(writer, "UIMinimumFontSize", new XIBObjectFloat(_minimumFontSize));
+    } else if (_minimumScaleFactor != -1.0f) {
+        AddOutputMember(writer, "UIMinimumScaleFactor", new XIBObjectFloat(_minimumScaleFactor));
+    } else {
+        AddBool(writer, "UIAdjustsFontSizeToFit", false);
```

Comment:
there is no default for adjustFontToFit, it is either false or true. depending on whether or not you set minimum font size or minimum scale. if set, it is true. otherwise false. both xib and storyboard uses this contract.

---
In reply to: [99473218](https://github.com/Microsoft/WinObjC/pull/1918#discussion_r99473218) [](ancestors = 99473218)